tests/integration/common/test_common.py

```diff
@@ -0,0 +1,90 @@
+from common.pull_ftp import migrate_from_ftp, reprocess_files
+from pytest import fixture
+from springer.repository import SpringerRepository
+from springer.sftp_service import SpringerSFTPService
+from structlog import get_logger
+
+
+@fixture
+def springer_empty_repo():
+    repo = SpringerRepository()
+    repo.delete_all()
+    yield repo
+
+
+def test_force_pull_from_sftp(springer_empty_repo):
+    with SpringerSFTPService() as sftp:
+        migrate_from_ftp(
+            sftp,
+            springer_empty_repo,
+            get_logger().bind(class_name="test_logger"),
+            **{
+                "params": {
+                    "excluded_dirs": [],
+                    "filenames_pull": {
+                        "enabled": True,
+                        "filenames": [
+                            "JHEP/ftp_PUB_19-01-29_20-02-10_JHEP.zip",
+                            "EPJC/ftp_PUB_19-02-06_16-01-13_EPJC_stripped.zip",
+                        ],
+                        "force_from_ftp": True,
+                    },
+                }
+            }
+        )
+        assert len(springer_empty_repo.find_all()) == 2
```

Comment:
let's assert with the exact expected values, not just the length - this might cover real bugs. This comments applies to the other places as well.